docs/book/src/SUMMARY.md

```diff
@@ -0,0 +1,14 @@
+# Summary
+
+[Introduction](introduction.md)
+
+- [Starting an xDSL project](starting.md)
+- [An xDSL program in examples](examples.md)
+- [Intermediate Representation (IR)](ir/index.md)
+    - [Dialects](ir/dialects.md)
+    - [Attributes](ir/attributes.md)
+    - [Regions](ir/regions.md)
+    - [Verifiers](ir/verifiers.md)
```

Comment:
```suggestion
    - [Attributes](ir/attributes.md)
    - [Operations](ir/operations.md)
    - [Regions](ir/regions.md)
    - [Verifiers](ir/verifiers.md)
    - [Dialects](ir/dialects.md)
```

Reply:
Oh also Traits

Reply:
Yes. I feel like traits could be introduced later, after rewriting, in a re-visit to introduce more complex concepts like traits/interfaces. But maybe there is not enough material to do this.

Reply:
Custom syntax also somewhere late